package/gluon-alt-esc-client/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-alt-esc-client
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
```

Comment:
This needs to be `include ../gluon.mk` with LEDE based builds. Same applies to the provider package.